pages/linux/mmcli.md

```diff
@@ -0,0 +1,12 @@
+# mmcli
+
+> Control and monitor the ModemManager.
+> More information: <https://www.freedesktop.org/software/ModemManager/man/latest/mmcli.1.html>.
+
+- List SMS messages available on the modem:
+
+`sudo mmcli --modem={{modem}} --messaging-list-sms`
+
+- Delete a message from the modem by path:
```

Comment:
```suggestion
- Delete a message from the modem specifying its path:
```
This wording is more common among the other pages.

Reply:
I didn't think about that. I searched a bit through the commands to confirm, but I think most tend _not_ to acknowledge their parameters at all. Could it be worth dropping that entirely?

```suggestion
- Delete a message from the modem:
```

![image](https://user-images.githubusercontent.com/22801583/116916190-49781d80-ac4d-11eb-96ba-f7a9fc2122b4.png)
> It seems most of the description that explicitly say "specify the path" do so because the command is setting an output path or similar.
 
![image](https://user-images.githubusercontent.com/22801583/116917124-8a246680-ac4e-11eb-9949-a90f35054ced.png)
> While some mention "specifying its path" or something similar, it's not very common.

![image](https://user-images.githubusercontent.com/22801583/116917543-12a30700-ac4f-11eb-8ae1-853bbf722e92.png)
> Looking at commands like 7z, adb, etc most don't even acknowledge the paramaters, or that a path is required, it's only inferred by the command itself.

Reply:
I wouldn't remove it entirely since we should try to keep the descriptions as simple as possible. I just think that `by path` sounds a little odd. However, I'm not an native speaker, so let's see what @bl-ue thinks.

Reply:
It's actually perfectly normal to say something like that.  I'd prefer a slight more verbose version, however:
```suggestion
- Delete a message from the modem by its path:
```
wdyt @SethFalco?

Reply:
Objectively speaking, I think @marchersimon has a point with "sounds a little odd". Even if mine's grammatically correct, it could be worth being more verbose if it aids in accessibility. (Baring in mind a lot of non-natives will be using the English version anyway.)

As a native speaker, I prefer my version, but ofc I'd say that, I wrote it. 🥇
I'm pretty comfortable with the terse versions, but think it could be worth using the initial suggestion if the explicitness makes things clearer.